### DISCUSSION TITLE
Slayer Log (KC Tracker)

### DISCUSSION THREAD

**Discussion by ghost** [runelite/runelite#15689] [Q&A] [answered]
Save us from having to carry an enchanted gem while killing large amounts of slayer monsters while off-task.

![image](https://user-images.githubusercontent.com/14812476/190866209-1ac06a94-e7b4-448b-afc5-ed7a3fa98232.png)

**Comment by Enriath** [comment, marked as answer]
Duplicate of #15227
(Same requirements to make it work)
